specification/advisor/resource-manager/Microsoft.Advisor/stable/2022-10-01/advisor.json

```diff
@@ -0,0 +1,1766 @@
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "category",
+            "modelAsString": true
+          }
+        },
+        "impact": {
+          "description": "The business impact of the recommendation.",
+          "enum": [
+            "High",
+            "Medium",
+            "Low"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "impact",
+            "modelAsString": true
+          }
+        },
+        "impactedField": {
+          "description": "The resource type identified by Advisor.",
+          "type": "string"
+        },
+        "lastUpdated": {
+          "format": "date-time",
+          "description": "The most recent time that Advisor checked the validity of the recommendation.",
+          "type": "string"
+        },
+        "shortDescription": {
+          "$ref": "#/definitions/ShortDescription",
+          "description": "A summary of the recommendation."
+        }
+      }
+    },
+    "scoreEntity": {
+      "description": "The details of Advisor Score",
+      "type": "object",
+      "properties": {
+        "date": {
+          "description": "The date score was calculated.",
+          "type": "string"
+        },
+        "score": {
+          "description": "The percentage score.",
+          "type": "number"
+        },
+        "consumptionUnits": {
+          "description": "The consumption units for the score.",
+          "type": "number"
+        },
+        "impactedResourceCount": {
+          "description": "The number of impacted resources.",
+          "type": "number"
+        },
+        "potentialScoreIncrease": {
+          "description": "The potential percentage increase in overall score at subscription level once all recommendations in this scope are implemented.",
+          "type": "number"
+        },
+        "categoryCount": {
+          "description": "The count of impacted categories.",
+          "type": "number",
+          "readOnly": true
+        }
+      }
+    },
+    "timeSeriesEntity": {
+      "description": "The historic data at different aggregation levels.",
+      "type": "array",
+      "items": {
+        "description": "The data from different aggregation levels.",
+        "type": "object",
+        "properties": {
+          "aggregationLevel": {
+            "description": "The aggregation level of the score. ",
+            "type": "string",
+            "enum": [
+              "week",
+              "day",
+              "month"
+            ],
+            "x-ms-enum": {
+              "name": "aggregated",
+              "modelAsString": true
+            }
+          },
+          "scoreHistory": {
+            "description": "The past score data",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/scoreEntity"
+            },
+            "x-ms-identifiers": []
+          }
+        }
+      },
+      "x-ms-identifiers": []
```

Comment:
Empty?

Reply:
@weidongxu-microsoft : was following the convention specified here : 
<img width="600" alt="image" src="https://user-images.githubusercontent.com/105078656/217043109-a26a6ff7-8160-4916-ab1e-a12ec013badc.png">